docs/0-zarf-overview.md

```diff
@@ -118,10 +118,11 @@ In the more complex use case, your package consists of updates for many apps/sys
 ## Why Use Zarf?
 
 - 💸 **Free and Open-Source.** Zarf will always be free to use and maintained by the open-source community.
+- ⭐️ **Zero Dependencies.** As a statically compiled binary, zarf has zero dependencies to run on any machine. 
```

Comment:
```suggestion
- ⭐️ **Zero Dependencies.** As a statically compiled binary, the Zarf CLI has zero dependencies to run on any machine. 
```
Just some more clarification since packages could have their own deps.